src/osemgrep/core/Cli_match.ml

```diff
@@ -0,0 +1,99 @@
+module OutT = Semgrep_output_v1_t
```

Comment:
please add a .mli

We usually use just a .ml when the file is mostly type definitions (because it's a bit annoying to copy paste all those type definitions), but for files having mostly functions, we usually prefer to have a .mli explaining how to use the module.

Reply:
done in [a7869a2](https://github.com/semgrep/semgrep/pull/9706/commits/a7869a28d7983febd689530478e839c9d02c6e5c) -- of course there could be even more documentation, and I didn't go into "how to use the module".